lib/katello/plugin.rb

```diff
@@ -272,4 +272,14 @@
 
   Katello::PermissionCreator.new(self).define
   add_all_permissions_to_default_roles
+
+  role 'Register hosts', [
+    :view_hostgroups, :view_activation_keys, :view_hosts,
+    :create_hosts, :edit_hosts, :destroy_hosts,
```

Comment:
I think `:view_content_hosts` and the rest are now merged into `hosts` permissions, that's why I didn't include them here. If you think they should be included and are still valid please ping me.